ethtx/decoders/decoder_service.py

```diff
@@ -153,7 +153,9 @@ def decode_transaction(self, chain_id: str, tx_hash: str) -> DecodedTransaction:
 
         used_semantics = self.semantic_decoder.repository.end_record()
         log.info(
-            "Semantics used in decoding %s: %s", tx_hash, ", ".join(used_semantics)
+            "Semantics used in decoding %s: %s",
+            tx_hash,
+            ", ".join(used_semantics) if isinstance(used_semantics, list) else "",
```

Comment:
This can only be a list or None, if it's none, there could be an information about this